app/validators/blocks/list_collector_validator.py

```diff
@@ -29,7 +29,10 @@ class ListCollectorValidator(BlockValidator, ValidateListCollectorQuestionsMixin
         "Multiple list collectors populate a list using different "
         "answer_ids in the add block"
     )
-    NON_SINGLE_REPEATING_BLOCKS_LIST_COLLECTOR = "List may only have one List Collector, if the List Collector features Repeating Blocks"
+    NON_SINGLE_REPEATING_BLOCKS_LIST_COLLECTOR = (
+        "List may only have one List Collector and List Collector Content, if "
+        "the List Collector Content or List Collector features Repeating Blocks"
+    )
```

Comment:
I probably missed this when it went in, why would we not allow this?. We should only now allow it if it is within the same section. If two collectors reference the same list in two different sections, we should allow it.

Reply:
I thought it made sense when reading this so that if you had two "people" list collectors and with different repeating blocks, adding people on the second list would leave the first list incomplete, and you'd need to go back for those repeating blocks, so it seems user friendly for the two sets of repeating blocks to be on the same list collector? Don't see why it wouldn't work with multiple though

Reply:
Ah I was thinking of it like, the additional list collector could only be off type content so they wouldn't allow adding.

Reply:
Ahh so limit to 1 normal list collector with repeating blocks but allow any number of content ones? don't see any harm in that

Reply:
> though I interpreted the comment above from the first review to mean we might not want this? was the original validation what was wanted?

Not sure I am following.

The original seem to only check that we don't have more than 1.

I think it should be: only 1 per list name per section. However, any amount of additional are allowed if they are content list collectors.

Reply:
Ah of course, the existing probably misses the section limit for content ones, but can have multiple in different sections and stick to a single normal + repeating one

Makes sense to me now 👍 agree with your verdict

Reply:
So, after reading all this, correct me if I'm wrong but my validation is checking now that **there is only 1 of either type per list name and per section**.

Reply:
> We need to enforce only 1 list per section when section summary is used (this might exist already)

Ok, I've added this check in section summary validation part of section validation now.

Reply:
I've gotten a bit lost, I think its worth adding a clear docstring wherever this is checked
I think the above alludes to having:

a given list to only have one list collector within a given section - (regardless of type I think?)

a list to only have one list collector with repeating blocks, if there are other list collector content blocks with repeating blocks that use it?
_____
So if I have a list collector of employees with some repeating blocks LC1
and a content block LCC1 with some more repeating questions, and another content block LCC2 with some more questions - I think that this is ok?

but if I had LC1 and then could also add to the list in LC2 with those questions, I think that would not be fine
_____
but im not sure thats exactly what is verified

Reply:
If we have a list name used twice but in different sections, why would we not allow it?.

Reply:
I think the idea behind not allowing it was if both of them were normal list collectors with repeating blocks
so if you had people LC + questions about bills (section 1)
then people LC + questions about dependents (section 2)
if you added a person in section 2, the repeating blocks for section 1 are then not complete, so you have to go back

which might be fine / an author choice, but I thought that slightly undesirable route might be why we wouldn't want multiple normal ones with repeating blocks for the same list

Reply:
As we discussed,  added new multiple list collectors validation on section level.